hw/ip/otp_ctrl/data/BUILD

```diff
@@ -140,7 +140,7 @@ otp_json(
                 "OWNER_SW_CFG_DIGEST": "0x0",
                 # Enable bootstrap. See `hardened_bool_t` in
                 # sw/device/lib/base/hardened.h.
-                "OWNER_SW_CFG_ROM_BOOTSTRAP_EN": "0x739",
+                "OWNER_SW_CFG_ROM_BOOTSTRAP_DIS": "0x1d4",
```

Comment:
Thanks. This is intentionally defined as a _magic_ value: OTP (as an IP) should probably not really depend on the arbitrary constants we chose for ROM/ROM_EXT and we have e2e tests to verify this. If we change our minds this would be a trivial update so I won't create a follow-up PR for now.